examples/explorer.py

```diff
@@ -8,7 +8,8 @@
 #
 # Thanks for using Enthought open source!
 
-""" A file explorer example. """
+""" A file explorer example. 
+    Note: This demo only works on the wx backend. """
```

Comment:
Ultra-Nit that can be ignored: For consistency in this code base, if there are more than one line, the closing `"""` should be in the next line on its own.